starters/svelte-kit-scss/src/lib/components/Counter.svelte

```diff
@@ -1,101 +1,48 @@
 <script lang="ts">
-  import { spring } from 'svelte/motion';
+  import { countStore } from '$lib/stores';
 
-  let count = 0;
+  const reset = (): void => countStore.set(0);
```

Comment:
stores should be used for shared state. otherwise we can just a regular variable in the component or a reactive statement if needed.
```
let count = 0
const increment = () => count += 1
const decrement = () => count -= 1
```
In Svelte assignments are reactive

Reply:
this is local state management.